warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__shapes_valid.sql

```diff
@@ -1,41 +1,52 @@
-WITH feed_guideline_index AS (
-    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+WITH guideline_index AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_quality__guideline_checks_index') }}
+    WHERE check = {{ shapes_valid() }}
 ),
 
 -- For this check we are only looking for errors related to shapes
-validation_fact_daily_feed_codes_shape_related AS (
-    SELECT * FROM {{ ref('fct_daily_schedule_feed_validation_notices') }}
-     WHERE code IN (
+shape_validation_notices AS (
+    SELECT *
+    FROM {{ ref('fct_daily_schedule_feed_validation_notices') }}
+    WHERE code IN (
             'decreasing_shape_distance',
             'equal_shape_distance_diff_coordinates',
             'equal_shape_distance_same_coordinates',
             'decreasing_or_equal_shape_distance'
             )
 ),
 
-shape_validation_notices_by_day AS (
+feed_shape_validation_notices AS (
     SELECT
         feed_key,
-        date,
-        SUM(total_notices) as validation_notices
-    FROM validation_fact_daily_feed_codes_shape_related
-    GROUP BY feed_key, date
+        SUM(total_notices) AS validation_notices
+    FROM {{ ref('fct_daily_schedule_feed_validation_notices') }}
+    GROUP BY feed_key
+),
+
+check_start AS (
+    SELECT MIN(date) AS first_check_date
+    FROM shape_validation_notices
 ),
 
 int_gtfs_quality__shapes_valid AS (
     SELECT
-        idx.date,
-        idx.feed_key,
-        {{ shapes_valid() }} AS check,
-        {{ accurate_service_data() }} AS feature,
+        idx.* EXCEPT(status),
         CASE
-            WHEN notices.validation_notices = 0 THEN {{ guidelines_pass_status() }}
-            WHEN notices.validation_notices > 0 THEN {{ guidelines_fail_status() }}
+            WHEN has_schedule_feed
+                THEN
+                    CASE
+                        WHEN validation_notices = 0 THEN {{ guidelines_pass_status() }}
```

Comment:
this can be a future todo item, but it might make sense to require a shapes.txt file in order to be eligible. We have a different check for "is there a shapes.txt file?"

Reply:
Added in https://github.com/cal-itp/data-infra/pull/2370/commits/19d724ec8a55ea8d47dab2492ace9ef699ec66a4 ✅ But doesn't really change outcomes, looks like basically everybody has shapes! 🎉